README.txt

```diff
@@ -146,6 +146,16 @@ $ ./bin/minetest
 - If you build a bare server, you don't need to have Irrlicht installed.
   In that case use -DIRRLICHT_SOURCE_DIR=/the/irrlicht/source
 
+With PostgreSQL Backend
+-----------------------
+The PostgreSQL backend does not build by default.
+
+In addition to the above dependencies, the PostgreSQL backend requires the following.
+# apt-get install libpq-dev postgresql-client postgresql-server-dev
```

Comment:
Or remove `PostgreSQL_backend.txt` since it doesn't provide much useful information other than this and some links that can easily be found with a web search.